app/tasks.py

```diff
@@ -135,7 +135,7 @@ def import_product_db(db: Session, batch_size: int = 1000):
             )
             item = normalize_product_fields(item)
             execute_result = db.execute(
-                Product.__table__.update()
+                update(Product)
```

Comment:
is this linked to typing ?

Reply:
Yes, this fixes typing, if I remember correctly mypy used to complain about `update()` method not being there because of old style of calling update